linux_os/guide/system/accounts/accounts-session/accounts_tmout/rule.yml

```diff
@@ -25,6 +25,8 @@ description: |-
     {{{- "or <tt>/etc/bashrc</tt>" if product in ["ol7", "rhel7"] }}}, e.g.
     <tt>/etc/profile.d/tmout.sh</tt> should read as follows:
     <pre>typeset -xr TMOUT={{{ xccdf_value("var_accounts_tmout") }}}</pre>
+    or
+    <pre>declare -xr TMOUT={{{ xccdf_value("var_accounts_tmout") }}}</pre>
```

Comment:
Maybe it would be good to inform the remediation preference is using typeset for wider compatibility?